utils/docker-compose/node-sdk/invoke.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict';
+
+const { FileSystemWallet, Gateway } = require('fabric-network');
+const fs = require('fs');
+const path = require('path');
+
+const ccpPath = path.resolve(__dirname, 'connection.json');
+const ccpJSON = fs.readFileSync(ccpPath, 'utf8');
+const ccp = JSON.parse(ccpJSON);
+
+async function main() {
+    try {
+        var user = process.argv[2]
```

Comment:
given that this is a very handy general purpose tool, it would be nice to do error-checking and/or have a -h/-? option with usage?

Reply:
I agree! I will address that in a followup commit